### PR TITLE
[OGC-954] update pvc output

### PR DESCRIPTION
**Pull request by zhaoyue722**
## What does this PR change?
* Add pvc name and storage class in the opencost allocation api response

## Does this PR relate to any other PRs?
* n/a

## How will this PR impact users?
* 

## Does this PR address any GitHub or Zendesk issues?
* Closes ...

## How was this PR tested?
* 

## Does this PR require changes to documentation?
* 

## Have you labeled this PR and its corresponding Issue as "next release" if it should be part of the next OpenCost release? If not, why not?
*

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -192,4 +192,4 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
 
-go 1.21.0
```
Comment (robertwenquan) [inline go.mod]: 这个别动吧，我们这个是可以考虑merge回社区去的。
只做最小改动会容易过一些

File: pkg/costmodel/allocation_helpers.go [hunk 2]
```diff
@@ -1938,6 +1942,7 @@ func applyPVCBytesRequested(pvcMap map[pvcKey]*pvc, resPVCBytesRequested []*prom
 	}
 }
 
+// query `avg(avg_over_time(pod_pvc_allocation{%s}[%s])) by (persistentvolume, persistentvolumeclaim, pod, namespace, %s)`
```
Comment (robertwenquan) [inline pkg/costmodel/allocation_helpers.go]: unused code

**Review by robertwenquan — COMMENTED** [review]
有一些没有用到的code和非必要改动
这个PR是可以考虑合并会开源社区的，做好更多的cleanup

File: go.mod (hunk 1 above)
Comment (zhaoyue722) [reply go.mod]: reverted back

File: pkg/costmodel/allocation_helpers.go (hunk 2 above)
Comment (zhaoyue722) [reply pkg/costmodel/allocation_helpers.go]: removed

**Review by robertwenquan — APPROVED** [review]

**Pull request merged**